File: src/main/java/homeWorkSeven/model/Water.java
package homeWorkSeven.model;

public class Water extends Product {
    private double price;
    private double volume;
    public Water(){

    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public double getVolume() {
        return volume;
    }

    public void setVolume(double volume) {
        this.volume = volume;
    }

    @Override
    public String getName() {
        return super.getName();
    }

    @Override
    public void setName(String name) {
        super.setName(name);
    }

    @Override
    public String toString() {
        return "Water{" +
                "price=" + price +
                ", volume=" + volume +
                ", name='" + name + '\'' +
                '}';
    }
}

File: src/main/java/homeWorkSeven/model/Water.cs
namespace homeWorkSeven{
    public class Water : Product
{
    private double price;
    private double volume;
    public Water(){}

    public double GetPrice() 
    {
        return price;
    }

    public void SetPrice(double price) 
    {
        this.price = price;
    }

    public double GetVolume() 
    {
        return volume;
    }

    public void SetVolume(double volume)
    {
        this.volume = volume;
    }

    public new string GetName()
    {
        return base.GetName();
    }

    public new void SetName(string name) 
    {
        base.SetName(name);
    }

    public new string ToString() 
    {
        return "Water{" +
                "name='" + name +
                ",  price=" + price+
                ", volume=" + volume + '\'' +
                '}'; 
    }    
}

}
